Userland/Libraries/LibWeb/Dump.cpp

```diff
@@ -45,863 +46,873 @@
 #include <LibWeb/SVG/SVGDecodedImageData.h>
 #include <stdio.h>
 
-namespace Web {
-
-static void indent(StringBuilder& builder, int levels)
+    namespace Web
 {
```

Comment:
What's going on with the massive whitespace-only (except like 10 lines) diff in the first commit? Contents of a namespace should not be indented by 4 spaces. Are you using clang-format-18 to format these changes?

Reply:
Unclear what happened there. Sorry. 

Fix coming up, with a fix for the build a well